Remotion/Web/Development.WebTesting/WebDriver/Factories/Chrome/ChromeBrowserFactory.cs

```diff
@@ -41,14 +42,16 @@ public ChromeBrowserFactory ([NotNull] IChromeConfiguration chromeConfiguration)
       _chromeConfiguration = chromeConfiguration;
     }
 
-    public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfrastructureConfiguration)
+    public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfrastructureConfiguration, DriverOptions options)

```

Comment:
What if we make the DriverOptions NotNull here, and use the TestInfrastructureConfiguration in the WebTestHelper to supply the correct value there? Would be a breaking change in the IBrowserFactory though, but im not sure if anyone uses this method currently. But i see the value in the current Approach. Delegating the Decisionmaking to MK